packages/env-tests/src/shared/attestation.ts

```diff
@@ -84,17 +85,18 @@ export async function requestAttestationsFromIssuers(
   attestations: AttestationsWrapper,
   phoneNumber: string,
   account: string,
-  pepper: string
+  pepper: string,
+  securityCode?: boolean
 ): Promise<RequestAttestationError[]> {
   return concurrentMap(5, attestationsToReveal, async (attestation) => {
     try {
       const revealRequest: AttestationRequest = {
         phoneNumber,
         account,
-        issuer: account,
+        issuer: attestation.issuer,
```

Comment:
wow big miss on my side 🙏

Not entirely sure how that has worked before?

Reply:
😅 no idea, the test accounts are generated on every run. Maybe it wasn't always that way and you could specify a mnemonic?